packages/slo-stat-bot/test/slo-stat-label.ts

```diff
@@ -0,0 +1,305 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {resolve} from 'path';
+import {Probot} from 'probot';
+import nock from 'nock';
+import * as fs from 'fs';
+import {expect} from 'chai';
```

Comment:
In general we don't use `expect` from chai at all and use the plain `assert` package.

Reply:
Oh I didn't realize that because the "auto-label" bot uses expect for the unit tests. But I can change it to assert.

Reply:
I see why assert is better to use in this scenario than expect because we want the current function to abort fast.